design/use_cases/multi-story-building_use_case.md

```diff
@@ -0,0 +1,13 @@
+# Multi-story Building Navigation (2D+)
+As a Robot user I want my robot to be able to navigate stairways or elevators to move to another portion of the multi-story building so that it can do something useful
+
+## More details
+- Why is this needed?
+   - Example: a delivery robot in an office building 
+
+- What is the expected user interaction?
+   - The user should be able to specify a stairways and elevators on a map for a robot to use or not use
```

Comment:
I think this "specification" should be more filled out -- you're probably providing an external action that can perform the "teleportation" i mentioned above, as well as specifying start/end points of teleportation. From an implementation standpoint, it also probably puts requirements on how we build the planners to make the aware of these locations/paths.

Reply:
Right now I'm just stating that we need a way to do it, which it sounds like you agree with.